migration_scripts/item_ball_refactor.py

```diff
@@ -0,0 +1,80 @@
+import glob
+import re
+import json
+
+# scan incs
+incs_to_check =    glob.glob('./data/scripts/*.inc') # all .incs in the script folder
+incs_to_check +=   glob.glob('./data/maps/*/scripts.inc') # all map scripts
+pories_to_check =  glob.glob('./data/scripts/*.pory') ## all .porys in the script folder
+pories_to_check += glob.glob('./data/maps/*/scripts.pory') # all map scripts
```

Comment:
Those need changing if you run it from a sub-folder right?

Reply:
Nah, I just tested it in a different Emerald and it seems to work, but somebody else should check on me on this.

Reply:
This is odd. Afaik `.` is the root folder so it shouldn't work? I'll test it tomorrow.

Reply:
Yep, just tested. It doesn't work. It needs to be changed to the parent directory.

Well, Idk. I guess a decision should be made here since you can run the script from the root folder `python sub-folder/script.py`. I guess it doesn't really matter but my I would probably move into the scripts folder to run it if I didn't know better.

Reply:
> Migration scripts are executed by running the following commands from the root directory of a developer's project.

```bash
chmod +x migration_scripts/*.py ;
python3 migration_scripts/*.py ;
```

these are the instructions in the README. Do these instructions fail?

OR did you do:

```bash
cd migration_scripts ;
chmod +x *.py ;
python3 *.py
```

Reply:
I understand that `python3 migration_scripts/*.py` works. I'm just saying that people will go into the sub-folder and run the script from there without reading the instructions. Which will not work.  

Btw is there a reason to add the executable permissions? It works fine without.